docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+  * [Compatibility & Supported ODS / API Versions](#compatibility-supported-ods-api-versions)
+* [Installation Instructions](#installation-instructions)
+  * [General Prerequisites](#general-prerequisites)
+* [Installation Instructions](#installation-instructions)
+  * [1\. Include Admin API in the ODS Docker Setup](#1-include-admin-api-in-the-ods-docker-setup)
+  * [2\. Relaunch the Docker Composition](#2-relaunch-the-docker-composition)
+  * [3\. Execute First-Time Configuration](#3-execute-first-time-configuration)
+
+# Before You Install
+
+This section provides general information you should review before installing the Ed-Fi ODS / API Admin API for v1.4.0.
+
+## Compatibility & Supported ODS / API Versions
+
+This version of the Admin API has been tested and can be installed for use with the Ed-Fi ODS / API v3.4 - 6.1. See the [Ed-Fi Technology Version Index](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875717/Ed-Fi+Technology+Version+Index) for more details.
+
+# Installation Instructions
+
+## General Prerequisites
+
+The following are required to install the Admin API:
+
+* The Admin API provides an interface to administer an Ed-Fi ODS / API. Understandably, you must have an instance of the Ed-Fi ODS / API v3.4 - 6.1 deployed and operational before you can use the Admin API. Tested configurations include on-premises installation via [binary installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100419/Getting+Started+-+Binary+Installation) or [source code installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100348/Getting+Started+-+Source+Code+Installation).
+* A SQL Server 2012 or higher, or Postgres 11 or higher database server (i.e., the same platform requirement applicable to your ODS / API).
+* A modern web browser such as Google Chrome, Mozilla Firefox, or Microsoft Edge is required to view live Swagger documentation. Internet Explorer 11 (a pre-installed browser on Windows Server) may load but may not function when using Admin API.
+
+# Installation Instructions
+
+Admin API is not included with the ODS-Docker solution by default, but can be hosted as part of that ecosystem.
+
+To install Admin API on Docker, first Install the [ODS / API Docker](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Docker) environment [following these instructions](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Docker+Deployment). Then, apply the below changes to the environment to introduce the Admin API.  Admin API does not support in-place upgrades from prior versions.  Please install a fresh copy of Admin API to upgrade from prior versions.
+
+## 1\. Include Admin API in the ODS Docker Setup
+
+### Docker Compose
+
+Add the following to your `docker-compose.yml`  file. This can be done either instead of or in addition to the `adminapp`  service.
+
+#### Admin API Application
+
+This service depends on the `pb-admin`  and subsequently `db-admin` services to run.
+
+**docker-compose.yml**
+
+```
```

Comment:
```suggestion
```yaml title="docker-compose.yml"
```

This will give YML syntax highlighting, and the code block will have `docker-compose.yml` as a header/title.